File: platforms/ios/vshop/farwolf.weex/farwolf.weex/module/WXFarwolfModule.m
//
//  WXFarwolfModule.m
//  Pods
//
//  Created by 郑江荣 on 2017/6/15.
//
//

#import "WXFarwolfModule.h"
#import "Weex.h"


@implementation WXFarwolfModule
@synthesize weexInstance;



WX_EXPORT_METHOD(@selector(getBaseDir))
WX_EXPORT_METHOD(@selector(setBaseDir:))
-(id)getBaseDir
{
    return [Weex getBaseDir];
}

-(void)setBaseDir:(NSString*)basedir
{
    [Weex setBaseDir:basedir];
}

@end

File: platforms/android/farwolf.weex/src/main/java/com/farwolf/weex/module/WXFarwolfModule.java
package com.farwolf.weex.module;

import com.farwolf.weex.util.Weex;
import com.taobao.weex.annotation.JSMethod;
import com.taobao.weex.common.WXModule;

/**
 * Created by zhengjiangrong on 2017/6/15.
 */

public class WXFarwolfModule extends WXModule {



    @JSMethod
    public void setBaseDir(String basedir)
    {
        Weex.basedir=basedir;
    }



    @JSMethod(uiThread = true)
    public String getBaseDir()
    {
        return Weex.basedir;
    }



}
